R/human_infection.R

```diff
@@ -171,15 +190,51 @@ calculate_infections <- function(
   infected
 }
 
-#' @title Calculate clinical infections
+#' @title Calculate patent infections (vivax only)
 #' @description
-#' Sample clinical infections from all infections
+#' Sample patent infections from all infections
```

Comment:
I don't see doc changes for this, could you commit those?

Reply:
Documentation added.